lib/more_core_extensions/core_ext/object/descendants.rb

```diff
@@ -0,0 +1,15 @@
+module MoreCoreExtensions
+  module Descendants
+    #
+    # Retrieve a descendant by its name
+    #
+    def descendant_from_name(desc_name)
```

Comment:
Since this is so similar `const_get`, I think `descendant_get` might be a better name.  @jrafanie @Fryguy any thoughts?